app/views/questions/_questions.html.erb

```diff
@@ -1,6 +1,7 @@
+<% notes = questions || @questions # accept local if present, default to instance %>
```

Comment:
Hmm, @cesswairimu i'm seeing this in the errors... maybe we have to be sure this is set upstream? 

```
ActionView::Template::Error:         ActionView::Template::Error: undefined local variable or method `questions' for #<#<Class:0x000055b41d5a1b18>:0x000055b41dbc7538>
        Did you mean?  @questions
            app/views/questions/_questions.html.erb:1:in 
```

Reply:
Oops , it should be <% questions = questions || @questions # accept local if present, default to instance %> ... Is it okay to change it here?

Reply:
@gyi2521 yes you can change it here and commit. @jywarren yes it should be best to set upstream and origin to their normal forks.
@gyi2521 you can do this by running this commands
`git remote remove origin`

Reply:
Hi @cesswairimu , the only part I merged is that when I made the second PR , it said  I had a conflict so resolved the conflict by deleting my code... The conflicted code was as following:
<div class="text-center"><%= will_paginate wikis, :renderer => BootstrapPagination::Rails if @paginated %></div> ... on app/views/wiki/_wikis.html.erb page.  Somehow,<div class="text-center"> didn't exist on the version I worked on... but everything should look okay now.

@jywarren I changed the <% notes = questions || @questions > to <% questions = questions || @questions> just now, and it looks like ALL checks have passed.  Is it ready to be merged FINALLY?  :-) 

Thank you both for your support!